Makefile.in

```diff
@@ -46,6 +46,11 @@ SYSROOT := $(INSTALL_DIR)/sysroot
 ENABLE_LIBSANITIZER ?= @enable_libsanitizer@
 QEMU_TARGETS ?= @qemu_targets@
 
+ENABLED_LANGUAGES ?= @WITH_LANGUAGES@
+ifeq ($(ENABLED_LANGUAGES),$(sgy))
```

Comment:
What's `$(sgy)`?

Reply:
if ENABLED_LANGUAGES is empty then unset it. It will be re-declared in stage 2s and use default.

if ENABLED_LANGUAGES  is `c,c++,go`, do nothing. It will be used in stage 2s.

Reply:
But, as @kito-cheng asked, what is `$(sgy)`?

Reply:
empty

> But, as @kito-cheng asked, what is `$(sgy)`?

Reply:
> empty
> 
> > But, as @kito-cheng asked, what is `$(sgy)`?

So why not simply `ifeq ($(ENABLED_LANGUAGES),)`?
Using `$(sgy)` seems unnecessary and just confusing?

Reply:
> > empty
> > > But, as @kito-cheng asked, what is `$(sgy)`?
> 
> So why not simply `ifeq ($(ENABLED_LANGUAGES),)`? Using `$(sgy)` seems unnecessary and just confusing?

Done